cfme/containers/provider/__init__.py

```diff
@@ -269,6 +270,18 @@ def num_image_registry(self):
     def num_image_registry_ui(self):
         return int(self.get_detail("Relationships", "Image Registries"))
 
+    def pods_per_ready_status(self):
+        """Grabing the Container Statuses Summary of the pods from API"""
+        #  TODO: Add later this logic to wrapanapi
```

Comment:
(+ @psav ) Indeed, I didn't forgot that, my next step will be to migrate all these functions to wrapanapi (this is not the only one, I have similar TODOS all over the framework)